src/game/savegame/savegame_bson.c

```diff
@@ -797,6 +800,34 @@ static bool Savegame_BSON_LoadLara(
     return true;
 }
 
+static bool SaveGame_BSON_LoadCurrentMusic(struct json_object_s *music_obj)
```

Comment:
I've noticed overall that this function will always return `true`.

Reply:
Hm yes but is that an issue? I didn't want to cause a savegame loading failure because of the music track only. Just a warning.

Reply:
Was thinking it might be cleaner just to make it `void`, but appreciate that goes against the common approach with the other functions.